x-pack/plugins/security/server/authentication/providers/saml.test.ts

```diff
@@ -0,0 +1,1010 @@
+/*
```

Comment:
__note:__ diff#7: removed test that checks that provider throws if `realm` isn't provided (it's optional in 7.x) and migrated tests that verify `acs` we pass in `prepare` and `invalidate`.